### ISSUE TITLE
Remove `slideMapProgressBarPlugin` from `configOverrides` in `skins/opencast/theme.json`

### ISSUE THREAD

**Issue by owi92** [polimediaupv/paella-skins#6]
Hi,
@oas777 suggested in [elan-ev/tobira#1196 (comment)](https://github.com/elan-ev/tobira/issues/1196#issuecomment-2203483164) that he would prefer a different color for the vertical lines of the slide map progress bar (or rather just disabling their change in color on hover).
If I understand it correctly, that appears to be overidden in `theme.json` which we copy in Tobira to match Opencast's paella integration, and it doesn't seem to be customizable in Tobira itself.

Is it possible to make it either customizable or otherwise disable it? Maybe it could be removed from the `configOverrides`, but I'm not sure what the best approach is here.

I also understand that the purpose of this `theme` is to have these styles fixed and easily shared between projects so I can see this issue being somewhat controversial.

**Comment by miesgre** [comment]
Hi @owi92!

We can remove the `slideMapProgressBarPlugin` from the`ovwrrides`. But then the colors from config.json will be applied.

It would not be the same color for everyone who uses the theme. If that's what you want, no problem. Please talk to Lukas/Olaf and tell me.

**Comment by oas777** [comment]
Thanks, @miesgre. To be honest, my suggestion to change the color/behavior of the vertical "dividers" is just a workaround. What I really want is for them to be reduced to the height of the bar they are dividing:
![image](https://github.com/user-attachments/assets/e811f6db-6dbe-49f7-bae1-8aa8ddc793da)
Admittedly, this is a design question, compared to
![image](https://github.com/user-attachments/assets/8cf8ae1b-8629-4ba1-b7c9-dbe9374af98e)

So could you please give me some background information on why the dividers are the way they are?

**Comment by miesgre** [comment]
What do you think?

<img width="619" alt="Captura de pantalla 2024-07-16 a las 8 08 15" src="https://github.com/user-attachments/assets/1ef8f66e-6731-4508-a8ce-f06835055915">

**Comment by oas777** [comment]
Visually, that's exactly what I was thinking of, thank you. But why are there still vertical lines (higher than the horizontal line) you are hiding? What is their use case? 
@owi92, @LukasKalbertodt, what do you think?

**Comment by miesgre** [comment]
Yes, it is the only thing I can do without having to modify the plugin.
In the skin I can only play with the colors. The idea is to put the lines the same color as the background and adjust their thickness.

I will select the same exact color. It was a quick test to see if that was what you wanted.

**Comment by oas777** [comment]
+1

**Comment by owi92** [comment]
@miesgre that was my initial idea as well, but I think that won't work with the hover style, right?
 I suppose that could also use the same color as the background, but then it wouldn't have any purpose.
So it would really be best if the marks were reduced to the height of the progress bar, as @oas777 suggested.

This was my mistake, I believe I should have opened this issue in the paella-slide-plugins repository instead.

**Comment by LukasKalbertodt** [comment]
I agree that just changing the bar color to the background color doesn't work, especially in cases where the background is semi-transparent (like in Tobira). And like, in [this comment](https://github.com/polimediaupv/paella-skins/issues/6#issuecomment-2230104014) I still clearly see the bars on top of the background. So I agree with Ole in that the plugin needs to be adjusted.

**Comment by owi92** [comment]
I think this can be closed with the latest updates.
